src/darksend.cpp

```diff
@@ -748,14 +748,12 @@ void CDarksendPool::ChargeRandomFees()
 void CDarksendPool::CheckTimeout()
 {
     // check mixing queue objects for timeouts
-    int c = 0;
     std::vector<CDarksendQueue>::iterator it = vecDarksendQueue.begin();
     while(it != vecDarksendQueue.end()) {
         if((*it).IsExpired()) {
-            LogPrint("privatesend", "CDarksendPool::CheckTimeout -- Removing expired queue entry: %d\n", c);
+            LogPrint("privatesend", "CDarksendPool::CheckTimeout -- Removing expired queue (%s)\n", (*it).ToString());
```

Comment:
No, expired entries (actual entries being mixed in some session) are removed at 786+. Here we remove expired mixing queues (which are kind of possible mixing sessions we could participate in if we would have required denominations and wouldn't be mixing already).